lib/pbench/test/functional/agent/cli/commands/test_config.py

```diff
@@ -18,7 +18,10 @@ def test_pbench_run_dir_existence(monkeypatch, agent_config, pbench_run):
     out, err, exitcode = pytest.helpers.capture(command)
     assert exitcode == 1
     assert b"" == out
-    assert b"[ERROR] pbench run directory does not exist," in err
+    assert (
+        f"[ERROR] the provided pbench run directory, {os.getenv('pbench_run')}, does not exist\n".encode()
```

Comment:
We know that the value of `pbench_run` should be, since we set it at line 16; therefore, we should be checking that the error message contains _that_, rather than checking that it contains what is _currently_ in the environment.

That is, the code should look something like this:
```
    my_pbench_run = f"{pbench_run}/test"
    monkeypatch.setenv("pbench_run", my_pbench_run)
...
    assert (
        f"[ERROR] the provided pbench run directory, {my_pbench_run}, does not exist\n".encode()
        == err
    )
```